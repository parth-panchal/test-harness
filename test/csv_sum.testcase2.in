Name,Age,Salary
John,25,50000
Alice,30,60000
Bob,22,45000
Eve,35,70000
