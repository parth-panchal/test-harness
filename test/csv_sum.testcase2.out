Sum of Age: 112.0
